### PR TITLE
Add Event documentation example

### PR DESCRIPTION
**Pull request by anildigital**

### CODE REVIEW

**Comment by anildigital** [comment]
@pitr-ch any feedback?

**Comment by pitr-ch** [comment]
@anildigital sorry, I got confused with new Github feature. I did comment shortly after you've opened the PR, but apparently it has to be confirmed to be seen by other readers.

File: lib/concurrent/atomic/event.rb [hunk 1]
```diff
@@ -13,6 +13,22 @@ module Concurrent
...
+  #     puts "event ocurred"
+  #   end
+  #
+  #   t2 = Thread.new do
+  #     puts "t2 calling set"
+  #     event.set
+  #   end
+  #
+  #   [t1, t2].each(&:join)
+  #
```
Comment (anildigital) [inline lib/concurrent/atomic/event.rb]: Added output. Could not find nicer way to add documentation example output in yard doc.

**Comment by anildigital** [comment]
@pitr-ch Added example.

**Comment by pitr-ch** [comment]
Thanks @anildigital I did one more minor change, see d9e8787. I'll wait for travis and merge.

**Pull request merged**